Seasonal/AprilFools/MeateorHunt.cs

```diff
@@ -0,0 +1,107 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreFarms.cs
+//cs_include Scripts/CoreStory.cs
+using RBot;
+public class MeateorHunt
+{
+    public ScriptInterface Bot => ScriptInterface.Instance;
+
+    public CoreBots Core => CoreBots.Instance;
+    public CoreStory Story = new CoreStory();
+
+    public void ScriptMain(ScriptInterface bot)
+    {
+        Core.SetOptions();
+
+        StoryLine();
+
+        Core.SetOptions(false);
+    }
+
+    public void StoryLine()
+    {
+        if (Core.isCompletedBefore(8628))
+        {
+            Core.Logger("You have already completed this storyline");
+            return;
+        }
+
+        Story.PreLoad();
+
+        //Defeat the Giant ChickenCow
+        Core.EquipClass(ClassType.Solo);
```

Comment:
KillQuest and variants will automatically equip SoloClass, so these bits are redundant